compressor/filters/css_default.py

```diff
@@ -104,3 +104,28 @@ def url_converter(self, matchobj):
 
     def src_converter(self, matchobj):
         return self._converter(matchobj, 2, "src=%s%s%s")
+
+
+class CssRelativeFilter(CssAbsoluteFilter):
+    """
+    Do similar to ``CssAbsoluteFilter`` URL processing
+    but replace ``settings.COMPRESS_URL`` prefix with '../../'.
+    """
+
+    def add_suffix(self, url):
```

Comment:
I've refactored the code. Now it's not inside the `.add_suffix()` but in a separate method.